bsd-user/ppc/target_arch_thread.h

```diff
@@ -40,10 +40,24 @@ static inline void target_thread_set_upcall(CPUPPCState *regs, abi_ulong entry,
     /* r3 = arg */
     regs->gpr[3] = arg;
     /* srr0 = start function entry */
+
 #if defined(TARGET_PPC64) && !defined(TARGET_ABI32)
-    get_user_ual(regs->nip, entry);
-    get_user_ual(regs->gpr[2], entry + sizeof(abi_ulong));
-    get_user_ual(regs->gpr[11], entry + 2 * sizeof(abi_ulong));
+
```

Comment:
There's also an extra newline here relative to other files, so if you're splitting things up tweaking it wouldn't be terrible....

Reply:
Reworked this section.